README.md

```diff
@@ -30,5 +30,12 @@ Recursively sanitize all files and folders in a given directory
 contao-console contao:proper-filenames:sanitize myfolder -r
 ```
 
+The extension only analyses files that are stored in Contao's DBFS (tl_files). The DBFS should be synchronised
+before the call - either via the Dataimanager in the backend or with the following console call:
```

Comment:
To be consistent with the rest of the Readme you should replace "Dataimanager" (typo btw) with `File manager` or `Files`.

Reply:
thx - wollte ich noch ändern...